### ISSUE TITLE
how to get the reported mAP?

### ISSUE THREAD

**Issue by Wuzimeng** [serend1p1ty/SeqNet#31]
Hi, I trained the model with bs 1 and lr 0.0006, but only got mAP = 85.79%, so I wonder how to approach the performance you mentioned in your paper? Do I need to fix any other configs?

**Comment by serend1p1ty** [comment]
Set batchsize=5 can reproduce the mAP reported in the paper.

If your GPU only supports batchsize=1, try to fix BatchNorm in training.

Example:

```python
def fix_bn(m):
    if isinstance(m, nn.BatchNorm1d) or isinstance(m, nn.BatchNorm2d):
        m.track_running_stats = False
model.apply(fix_bn)
```

**Comment by Wuzimeng** [comment]
Thanks a lot. I tried this modification but only got mAP around 85% with bs = 1/2/3/4, any other suggestions?

**Comment by serend1p1ty** [comment]
If you use batchsize=4, you needn't fix BatchNorm, as the batchsize is big enough. According to my experience, the performance gap between batchsize=4 and batchsize=5 is within 1% (If I remember correctly).

**Comment by Wuzimeng** [comment]
Thanks, finally I made it. After changing the version, I got the desired result. I think the misalignment might be because I modified code `loss_oim = F.cross_entropy(projected, label, ignore_index=5554)` and used a high pytorch version.
